spec/components/notification_component_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe NotificationComponent, type: :component do
+  let(:content) { 'This is content' }
+  let(:style) { 'notice' }
+  let(:dismiss) { true }
+  let(:background) { false }
+  let(:alert) { false }
+  let!(:inline_component) { render_inline(NotificationComponent.new(content: content, style: style, dismiss: dismiss, background: background, alert: alert)) }
```

Comment:
Any chance of this being split into multiple lines?

```ruby
  let!(:inline_component) { render_inline(NotificationComponent.new(
                                                                    content: content,
                                                                    style: style,
```
etc. You can let rubocop sort out the indentation.

Reply:
Done!